pubspec.yaml

```diff
@@ -11,7 +11,7 @@ environment:
   flutter: ">=1.20.0"
 
 dependencies:
-  http: ^0.13.3
+  http: ^1.0.0
```

Comment:
the http 1.0.0 [Requires Dart 3.0 or later.](https://pub.dev/packages/http/versions/1.0.0/changelog) and our current SDK support Dart '>=2.12.0 <4.0.0' so wether we need to exclude Dart 2.x.x or keep using http ^0.13.3

Reply:
I reset the version to ^0.13.3 to keep support of dart 2.x.x. We won't have the max pub points but we don't have choice. Let's see that once invisible DC is released.